tests/test_expressions.py

```diff
@@ -258,6 +258,14 @@ def test_replace_tables(self):
             'SELECT * FROM "my-project"."example"."table" /* example.table */',
         )
 
+        self.assertEqual(
+            exp.replace_tables(
+                parse_one("select * from example.table /* sqlglot.meta preserve_reference */"),
```

Comment:
what about 

sqlglot.meta replace=False

Reply:
I thought about this as well. There are two reasons I went with `preserve_reference`:

1. We don't have logic to convert `"false"` and `"true"` to booleans in `Expression.add_comment` right now
2. Something like `replace` seemed a bit generic, whereas `preserve_reference` felt more explicit

None of these are super important, so happy to refactor if you prefer it. I guess if we went with your suggestion we'd also implement (1) to avoid comparing strings?

Reply:
i prefer replace because it matches the name

Reply:
Refactored, please take another look.